xrdp/xrdp_wm.c

```diff
@@ -29,7 +29,10 @@
 #include "log.h"
 #include "string_calls.h"
 
-
+/* Code values used in 'xrdp_mm->code=' settings */
+#define XVNC_SESSION_CODE 0
+#define XRDP_SESSION_CODE 10
+#define XORG_SESSION_CODE 20
 
```

Comment:
Could you move the above definitions into xrdp_types.h and combines them with the similar ones in xrdp_mm.c?